### ISSUE TITLE
Include information on vulnerable tech

### ISSUE THREAD

**Issue by double16** [enthec/webappanalyzer#20]
**Describe the solution you'd like**
Included with each tech found, indicate if it's vulnerable and a link to the vulnerability such as a CVE. Expose this data in such a way that another tool, such as ZAP, can leverage it.

**Describe alternatives you've considered**
Searching using other tools.

**Comment by psiinon** [comment]
Or just if the relevant version is no longer supported?

**Comment by kingthorin** [comment]
You said CWE but I think you meant CVE?

**Comment by double16** [comment]
> You said CWE but I think you meant CVE?

lol, yeah. Not enough coffee yet. ☕

**Comment by antoniojtorres** [comment]
I agree that it would make more sense as an integration alongside (and not in) this package. It is otherwise too different in the nature of how the data is collected and the cadence at which it needs to be collected.

**Comment by enthec-opensource** [comment]
Been thinking about this and the best solution I can think of is adding as many cpes as possible.

Reading about the cpe 2.3 specification: https://cpe.mitre.org/specification/ all that matters is vendor and product name (so we could just save those tbh) and then with whatever version webappanalyzer has detected create the string cpe:2.3:a:{vendor}:{product}:{version}:*:*:*:*:*:*:* (or even just without the version cpe:2.3:a:{vendor}:{product}:*:*:*:*:*:*:*:*) to find all the cves linked.

For example, lets say we have detected drupal version 5.10, the final string would be cpe:2.3:a:drupal:drupal:5.10:*:*:*:*:*:*:* which we can search on the nist api and find all linked CVEs: `https://services.nvd.nist.gov/rest/json/cves/2.0?cpeName=cpe:2.3:a:drupal:drupal:5.10:*:*:*:*:*:*:*` and leave that task to whatever implementation is working with this repo, because there is just no way to keep all the updated cves and cwes in real time.
